vector-im/element-android
# [Location sharing] - Small improvements of UI for live (PSG-613)

## Type of change


## Content


This contains some fixes on the UX for live location sharing message in timeline. See #6607 for more info.

## Motivation and context


Closes #6607 

## Screenshots / GIFs





<img src="https://user-images.githubusercontent.com/46314705/179993629-312310d2-48b4-4063-9147-91c44fa0d0ad.gif" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/179993661-dcea5ca5-ec83-4483-87f9-b3de1b6d0d3b.gif" width=300 />

## Tests



- Go to a room
- Start a live location share
- Check the loading state item is correct
- Check the active state item is correct
- End the live
- Check the ended state item is correct
- Reply to the live
- Check the message preview is correct
- Switch dark/light theme if you can
- Re-do the previous steps

## Tested devices

- OS version(s): Android 11

## Checklist
